docs/ddl-of-carbondata.md

```diff
@@ -474,7 +475,22 @@ CarbonData DDL statements are documented here,which includes:
      be later viewed in table description for reference.
 
      ```
-       TBLPROPERTIES('BAD_RECORD_PATH'='/opt/badrecords'')
+       TBLPROPERTIES('BAD_RECORD_PATH'='/opt/badrecords')
+     ```
+     
+   - ##### Load minimum data size
+     This property determines whether to enable node minumun input data size allocation strategy 
+     for data loading.It will make sure that the node load the minimum amount of data there by 
+     reducing number of carbondata files. This property is useful if the size of the input data 
+     files are very small, like 1MB to 256MB. And This property can also be specified 
+     in the load option, the property value only int value is supported.
+
+     ```
+       TBLPROPERTIES('LOAD_MIN_SIZE_INMB'='256 MB')
```

Comment:
I think we can remove this and only support '256' since the property name already contains 'INMB', this will make the code simple.

Reply:
Has been modified based on the review